source/horoscope/css/horoscope.css

```diff
@@ -211,10 +211,25 @@ input[type="button"] {
     border-radius: 4px;
     cursor: pointer;
     font-size: clamp(15px, 1.5vw, 30px);
+}
+
+/* submit button */
+#submit {
```

Comment:
I don't think the submit button needs to be on the horoscopes page now since the "submit" is done by the go on the landing page.